content/en/docs/reference/config/istio.operator.v1alpha1/index.html

```diff
@@ -138,10 +161,10 @@ <h2 id="IstioOperatorSpec">IstioOperatorSpec</h2>
 <td><code>values</code></td>
 <td><code><a href="#TypeMapStringInterface">TypeMapStringInterface</a></code></td>
 <td>
-<p>Overrides for default values.yaml. This is a validated pass-through to Helm templates.
-See the Helm installation options for schema details: https://istio.io/v1.5/docs/reference/config/installation-options/.
-Anything that is available in IstioOperatorSpec should be set above rather than using the passthrough. This
-includes Kubernetes resource settings for components in KubernetesResourcesSpec.</p>
+<p>Overrides for default <code>values.yaml</code>. This is a validated pass-through to Helm templates.
+See the <a href="/v1.5/docs/reference/config/installation-options/">Helm installation options</a> for schema details.
```

Comment:
```suggestion
See the Helm installation options at `https://istio.io/v1.5/docs/reference/config/installation-options/` for schema details.
```

Reply:
Let's just avoid the problem by changing it like this for now. WDYT?

Reply:
OK. Maybe I have something that will work doing some quick sed reading.